docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -0,0 +1,91 @@
+# Required Software
+
+1. [Visual Studio 2015 with Update 1](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 1_.
+2. Visual Studio 2015 Extensibility Tools. If you already installed Visual Studio, choose "Modify" from the Programs and Features control panel, and check "Visual Studio Extensibility".
+
+# Getting the Code
+
+1. Clone https://github.com/dotnet/roslyn
+2. Run the "Developer Command Prompt for VS2015" from your start menu.
+3. Run `Restore.cmd` in the command prompt to restore NuGet packages.
+4. Due to [Issue #5876](https://github.com/dotnet/roslyn/issues/5876), you should build on the command line before opening in Visual Studio. Run `msbuild /v:m /m Roslyn.sln`
```

Comment:
@diryboy: chatting with @agocke we agreed that it's probably best to leave things how they are. cibuild will do that for validation, but locally we won't try switching things out to avoid surprising you, and causing all sorts of headaches which would occur by that switching. I think it'd be possible to do, but it won't be something I'll sleep well with.

(Also to be honest I was unaware of the cibuild behavior until you mentioned it. I'm terrified, but not terrified enough to change it.)